src/Resources/views/storefront/page/checkout/_page.html.twig

```diff
@@ -34,12 +34,13 @@
         "refreshMessage":'myparcel.shipping_selector.refreshMessage' | trans,
     } %}
 
+    {{ dump(page.extensions) }}
```

Comment:
Ik denk date deze nog is blijven staan van een andere keer en nu hier niet bij hoort, net als de verandering verderop